subprojects/libzmq.wrap

```diff
@@ -0,0 +1,10 @@
+[wrap-file]
+directory = libzmq-4.3.5
+source_url = https://github.com/zeromq/libzmq/archive/refs/tags/v4.3.5.tar.gz
+source_filename = libzmq-4.3.5.tar.gz
+source_hash = 6c972d1e6a91a0ecd79c3236f04cf0126f2f4dfbbad407d72b4606a7ba93f9c6
+patch_directory = libzmq
+diff_files = libzmq/fix-mingw-afunixh.diff
```

Comment:
WrapDB has a policy that things in it may not have functional changes, only build system updates. Weird that there is not a test for this.

Reply:
There is, but I added an exception.

Reply:
In any case it is not permitted. We can't really have any functionality delta because we don't have the manpower to keep maintaining them and if anything goes wrong, then it's on us.

Reply:
Alright - then I will revert those changes and wait until a new version is released